.github/workflows/docker-build-push.yml

```diff
@@ -20,10 +20,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Build & test
-        working-directory: "${{ matrix.service_name }}"
+        #working-directory: "${{ matrix.service_name }}"
```

Comment:
Is that what caused the problem yesterday? If we don't need that, and it causes issues... Maybe it is better to delete it then comment out?

Reply:
done. pls resolve